components/chassis.py

```diff
@@ -396,7 +417,7 @@ def get_tilt_rate(self) -> float:
 
     @feedback
     def get_drive_current(self) -> float:
-        return sum(abs(x.drive.getStatorCurrent()) for x in self.modules)
+        return sum(abs(x.drive.get_stator_current().value) for x in self.modules)
```

Comment:
Eep, this is a code smell. The `Chassis` component should never have been reaching into the `SwerveModule` internals in the first place.

nit: add a `SwerveModule.get_drive_current() -> float` method